src/game/PetAI.cpp

```diff
@@ -335,6 +335,16 @@ void PetAI::UpdateAI(const uint32 diff)
             return;
         }
 
+        if (!m_creature->HasInArc(2 * M_PI_F / 3, victim))
+        {
+	        m_creature->SetInFront(victim);
```

Comment:
This is how it looks for me, and no, they are not tabs. I only work with spaces (It's by default 4 spaces per tab press in my settings, no actual tabs are added)
http://imgur.com/fLUVJIe - Did git maybe change something? Is there a setting?

Sorry for the nooby questions^^ Learning this is harder than c++